### DISCUSSION TITLE
Caching in Oqtane

### DISCUSSION THREAD

**Discussion by pnicosia80** [oqtane/oqtane.framework#4923] [Q&A]
Hi all, I have to cache for a few hours calls to api services that fill some dropdowns. What is better to use in oqtane?
Thanks
Have a nice day!!

**Comment by leigh-pointer** [comment]
@pnicosia80 have a look at the HTMLText service ServerHtmlTextService.cs in the oqtane.server project, modules folder. There you will find a good implementation.
